### PR TITLE
Rust vscode blog post

### PR DESCRIPTION
**Pull request by cyplo**

### CODE REVIEW

File: _posts/2017-11-26-rust-vscode.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+While on Linux VSCode with the Rust plugin seems to work more or less out of the box, on a Mac I needed to spend some time configuring it.
+
+First things first though, let's start by installing Rust version manager, rustup.
+
+```
+curl https://sh.rustup.rs -sSf | sh
+```
+
+We will be using nightly version of rust as to have one version that can compile all of our tools. This is mostly due to `clippy` requiring a nightly compiler.
```
Comment (Gryff) [inline _posts/2017-11-26-rust-vscode.md]: maybe make `clippy` a link? I had to look up what it was :)

**Review by Gryff — APPROVED** [review]

File: _posts/2017-11-26-rust-vscode.md (hunk 1 above)
Comment (cyplo) [reply _posts/2017-11-26-rust-vscode.md]: Good idea :)

File: _posts/2017-11-26-rust-vscode.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+For a more wholesome experience, please have some tools as well:
+
+```
+cargo install clippy rustfmt rustsym
+```
+
+Now finally, for the VSCode itself, press `cmd-p` and `ext install vscode-rust`. I'm using the new `Rust` extension as `Rusty Code` has been discontinued.
+
+If you're lucky - that's it, you should have working completion and highlighting in Rust files.
```
Comment (samanthalcohen) [inline _posts/2017-11-26-rust-vscode.md]: might be worth adding a line before telling them to open up a rust file in vs-code

**Comment by cyplo** [comment]
Need to add info on building and debugging

**Pull request merged**